src/content/pages/speakers-dinner.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Speakers' Dinner
+url: /speakers-dinner
+subtitle:
+  We're happy to invite all speakers to our special Speakers' Dinner!
+---
+
+# Speaker's Dinner
+We are excited to announce a special Speakers' Dinner! This exclusive event is for speakers only, providing an opportunity to meet and mingle before the main conference days.
+
+The dinner will feature multiple courses, and you can choose from a variety of alcoholic and non-alcoholic beverages. We also offer a variety of options to accommodate different dietary preferences, including vegetarian, vegan, and gluten-free.
+
+# Where?
+The Speakers' Dinner will be at the Mánes restaurant on the Vltava riverfront in Prague, featuring creative functionalist architecture and panoramic views of the Vltava River, the Dancing House, and the National Theatre.
```

Comment:
> at the Mánes restaurant
Might it be a good idea to link the URL to the restaurant?

Reply:
added ✅